test/Microsoft.ML.TestFramework/DataPipe/TestDataPipeBase.cs

```diff
@@ -1096,23 +1096,6 @@ protected Func<bool> GetColumnComparer(Row r1, Row r2, int col, ColumnType type,
                     return GetComparerVec<RowId>(r1, r2, col, size, (x, y) => x.Equals(y));
             }
 
-#if !CORECLR // REVIEW: Port Picture type to CoreTLC.
-            if (type is PictureType)
```

Comment:
I think this could have been ported from `PictureType` to `ImageType`, if I understand the problem correctly? Would require a reference to `ImageAnalytics`, though.

Reply:
Currently we are only building the project with `CORECLR` defined. So this is dead code that can be deleted.

Let's just delete it for now. If @Ivanidzo4ka thinks this is important, we can open a new issue to add image support here. Thoughts @Ivanidzo4ka?

Reply:
Feel free to delete. Some legacy code which we don't care about anymore